internal/commands/result.go

```diff
@@ -2061,6 +2081,28 @@ func parseContainersSonar(result *wrappers.ScanResult) wrappers.SonarLocation {
 	return auxLocation
 }
 
+func parseSscsSonar(result *wrappers.ScanResult) wrappers.SonarIssues {
+	sonarIssue := initSonarIssue(result)
+
+	// overriding ruleID set by default in initSonarIssue
+	if result.ScanResultData.RuleID != nil {
+		sonarIssue.RuleID = *result.ScanResultData.RuleID
+	}
+
+	sonarIssue.PrimaryLocation.FilePath = result.ScanResultData.Filename
+	if result.ScanResultData.Snippet != "" {
+		sonarIssue.PrimaryLocation.Message = fmt.Sprintf("%s : %s", result.ScanResultData.Snippet, result.Description)
+	} else {
+		sonarIssue.PrimaryLocation.Message = result.Description
+	}
+	var textRange wrappers.SonarTextRange
+	textRange.StartColumn = 1
+	textRange.EndColumn = 2
```

Comment:
StartColumn and EndColumn will always be 1 and 2?

Reply:
We don't have information about the Start and End Column. I followed the approach already existing for SCA and Containers. KICS is always 0 and 1 instead of 1 and 2 as well.